### DISCUSSION TITLE
SmartThings can now set up Matter devices on the iPhone

### DISCUSSION THREAD

**Discussion by lboue** [mfucci/node-matter#188] [Matter compatible devices / controllers]
### Samsung SmartThings
[SmartThings can now set up Matter devices on the iPhone](https://www.theverge.com/2023/1/13/23553413/smartthings-can-now-set-up-matter-devices-on-the-iphone) (Jan 13, 2023 Version 1.6.94)

Samsung has released the update for its [SmartThings iOS app](https://apps.apple.com/us/app/smartthings/id1222822904) that lets you add Matter-compatible devices to your home. Now iPhone owners have another option alongside Apple’s Home app to set up Matter devices. Once the Matter device is set up it should be controllable by any compatible app, such as Google Home, Apple Home, and SmartThings.Of course, there are still very few Matter devices available, with just a handful of companies shipping or updating existing devices to support the standard just yet. But that will be changing soon and now if you’ve got a SmartThings home and an iPhone, you’re ready for when they arrive.

**Comment by Apollon77** [comment]
@lboue if you have smartthings Then just try it ... install github version of node-matter, run "npm run build" and try t out ... if it is not working post the debug log as issue :-)

**Comment by JimBuzbee** [comment]
I just gave Smarthings a go and failed.  The process of adding a Matter device started after I scanned the QR code, but ultimately I think that there was a Smarthings App crash. First I get a warning about a non-certified device, then a few seconds after I say "continue" I get kicked back a couple of screens to the QR scan screen. I'll create an issue with log attached.

**Reply by Apollon77** [reply]
is it just the App or do you need more?

**Comment by JimBuzbee** [comment]
I believe you need a SmartThings basestation in order for the App to function.

**Comment by Hedda** [comment]
> > is it just the App or do you need more?
> 
> I believe you need a SmartThings basestation in order for the App to function.

Note! You need to have a compatible SmartThings Hub V2/V3 (STH-ETH series from 2015) and later as those will act as a Matter Controller. 

See: -> https://support.smartthings.com/hc/en-us/articles/11219700390804-SmartThings-x-Matter-Integration-

* Samsung announced upcoming "Samsung SmartThings Station" will also support Matter over Thread radio (Thread Border Router).
* "Aeotec Smart Home Hub" (rebranded Samsung SmartThings Hub v3) will have gotten an update that supports Matter over Wi-Fi.
* "Samsung SmartThings Hub v3" (ETH-ETH-300 from 2018) has been updated to support Matter over Wi-Fi.
* "Samsung SmartThings Hub v2" (ETH-ETH-250 from 2016) has been updated to support Matter over Wi-Fi.
* "Samsung SmartThings Hub v2" (ETH-ETH-200 from 2015) has been updated to support Matter over Wi-Fi.
* "SmartThings Family Hub" smart fridge might or might not get updated to support Matter at a later date.
* "SmartThings Hub v1" (F-H-ETH-001 and STH-ETH-001) has been discontinued and will not get updated to support Matter.
* "Samsung SmartThings Link" (USB dongle) for Nvidia Shield has been discontinued and will not get updated to support Matter.

PS: Sadly Samsung has said that they will not bridge connected Zigbee and Z-Wave devices to Matter (e.i. only ecosystem lock-in so far).

https://www.theverge.com/2022/10/13/23400462/samsung-smartthings-matter-upgrade-hubs

https://www.androidpolice.com/samsung-smartthings-control-matter-compatible-devices/
